src/scicat_configuration.py

```diff
@@ -237,12 +238,23 @@ class SciCatOptions:
     timeout: int = 0
     stream: bool = True
     verify: bool = False
+    urls: dict = field(default_factory=dict)
 
     @classmethod
     def from_configurations(cls, config: dict) -> "SciCatOptions":
         """Create SciCatOptions from a dictionary."""
         options = cls(**config)
-        options.headers = {"Authorization": f"Bearer {options.token}"}
+        options.host = options.host.removesuffix('/') + "/"
+        options.headers = {
+            **options.headers,
+            **{"Authorization": f"Bearer {options.token}"}
+        }
+        options.urls = {
+            "datasets" : urljoin(options.host, "datasets"),
+            "proposals" : urljoin(options.host, "proposals"),
+            "origdatablocks" : urljoin(options.host, "origdatablocks"),
+            "instruments": urljoin(options.host, "instruments"),
+        }
```

Comment:
These 4 endpoints don't have to be configurable from the config file, no...?
Can I turn them to be a ``property`` instead?